src/main/java/com/faforever/client/logging/analysis/AnalysisResult.java

```diff
@@ -0,0 +1,11 @@
+package com.faforever.client.logging.analysis;
+
+import com.faforever.client.notification.Action;
+
+import java.util.Map;
+
+public record AnalysisResult(Map<String, Action> result) {
+  public boolean isOk() {
+    return result.isEmpty();
+  }
+}
```

Comment:
It seems like a little overkill to have an object that just wraps a map

Reply:
fixed